### PR TITLE
Fix wig wag model and textures

### PR DESCRIPTION
**Pull request by RavenholmZombie**
(Thanks Kreative for the $40!)

Updates the Wigwag texture to be more in-line with their real world counterparts.

### CODE REVIEW

**Comment by RavenholmZombie** [comment]
For @CSX8600 


The finalized Vertical Wig Wag block will need to match the image above as much as possible, I've tried to add the counterweight piece at the bottom of the swinging arm, but it crashes the game. There's also the current issue with its animation being broken.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Sep 14 12:28:28 PDT 2015
+#Fri May 05 14:33:29 EDT 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-bin.zip
```
Comment (CSX8600) [inline gradle/wrapper/gradle-wrapper.properties]: plz no

File: src/main/java/com/clussmanproductions/trafficcontrol/blocks/VerticalWigWagRenderer.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
+package com.clussmanproductions.trafficcontrol.blocks;
```
Comment (CSX8600) [inline src/main/java/com/clussmanproductions/trafficcontrol/blocks/VerticalWigWagRenderer.java]: This should go in the `tileentity/render` package

File: src/main/java/com/clussmanproductions/trafficcontrol/tileentity/render/RendererWigWag.java [hunk 3]
```diff
@@ -22,8 +22,9 @@
 public class RendererWigWag extends TileEntitySpecialRenderer<WigWagTileEntity> {
 
 	ResourceLocation genericTexture = new ResourceLocation(ModTrafficControl.MODID + ":textures/blocks/generic.png");
-	ResourceLocation blackTexture = new ResourceLocation(ModTrafficControl.MODID + ":textures/blocks/black.png");
+	ResourceLocation blackTexture = new ResourceLocation(ModTrafficControl.MODID + ":textures/blocks/wigwag.png");
```
Comment (CSX8600) [inline src/main/java/com/clussmanproductions/trafficcontrol/tileentity/render/RendererWigWag.java]: Rename `blackTexture` to `wigWagTexture` or something to more accurately represent what this is.

**Review by CSX8600 — COMMENTED** [review]
Please also undo all of the gradle changes as that will conflict directly with updated gradle info

File: src/main/java/com/clussmanproductions/trafficcontrol/tileentity/render/RendererWigWag.java (hunk 3 above)
Comment (RavenholmZombie) [reply src/main/java/com/clussmanproductions/trafficcontrol/tileentity/render/RendererWigWag.java]: Fixed

File: src/main/java/com/clussmanproductions/trafficcontrol/blocks/VerticalWigWagRenderer.java (hunk 2 above)
Comment (RavenholmZombie) [reply src/main/java/com/clussmanproductions/trafficcontrol/blocks/VerticalWigWagRenderer.java]: Fixed

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (RavenholmZombie) [reply gradle/wrapper/gradle-wrapper.properties]: plz yes
(fixed lol, I think)

**Review by CSX8600 — APPROVED** [review]

**Pull request merged**